test/ci/buildkite/ngtf-ie-ubuntu.yml

```diff
@@ -12,6 +12,16 @@
       queue: ${BUILDKITE_AGENT_META_DATA_QUEUE}
       name: ${BUILDKITE_AGENT_META_DATA_NAME}
 
+  - wait
+  - command: |
+      NGRAPH_TF_BACKEND=IE:CPU PYTHONPATH=`pwd` python3 test/ci/buildkite/test_runner.py \
```

Comment:
`NGRAPH_TF_BACKEND` should be set to CPU here. The format for backend string is "plugin:device".

Reply:
If we set it to CPU for IE build, some tests will fail. e.g. Device with "" name is not registered in the InferenceEngine